x/evidence/keeper/infraction.go

```diff
@@ -119,4 +119,5 @@ func (k Keeper) HandleEquivocationEvidence(ctx sdk.Context, evidence *types.Equi
 
 	k.slashingKeeper.JailUntil(ctx, consAddr, types.DoubleSignJailEndTime)
 	k.slashingKeeper.Tombstone(ctx, consAddr)
+	k.SetEvidence(ctx, evidence)
```

Comment:
So we're duplicating TM evidences in state, correct? What do you think of https://github.com/cosmos/cosmos-sdk/issues/8360#issuecomment-762329075, to keep state lean?

Reply:
I feel it's better to have consistency across evidence handling(all being present in store), and being able to query it using evidence queries. wdyt?

Reply:
ok, it seems people generally agree about this duplication in #8360.